src/IrcClient/IrcMessageData.cs

```diff
@@ -141,6 +145,28 @@ public class IrcMessageData
         }
 
         /// <summary>
+        /// Gets the raw tags sent by the server
+        /// </summary>
+        public string RawTags
```

Comment:
what is the purpose of keeping unparsed tags, is there some use-case? RawMessage contains already all information if the lib user wants to parse something differently.